tests/integration/api_packages_test.go

```diff
@@ -403,3 +426,25 @@ func TestPackageCleanup(t *testing.T) {
 		}
 	})
 }
+
+// This method removes all package related data (database and files)
+func removeAllPackageData(t *testing.T) {
+	assert.NoError(t, db.DeleteAllRecords("package"))
+	assert.NoError(t, db.DeleteAllRecords("package_version"))
+	assert.NoError(t, db.DeleteAllRecords("package_file"))
+	assert.NoError(t, db.DeleteAllRecords("package_property"))
+	assert.NoError(t, db.DeleteAllRecords("package_blob_upload"))
+	assert.NoError(t, db.DeleteAllRecords("package_cleanup_rule"))
+
+	duration, _ := time.ParseDuration("-10h")
+
+	pbs, err := packages_model.FindExpiredUnreferencedBlobs(db.DefaultContext, duration)
+	assert.NoError(t, err)
+
+	contentStore := packages_module.NewContentStore()
+	for _, pb := range pbs {
+		contentStore.Delete(packages_module.BlobHash256Key(pb.HashSHA256))
+	}
```

Comment:
Why not ensure it is empty in prepareTestEnv?

Reply:
I did not want to add a burden to all other tests. Do you think it would be better placed there?

Reply:
I think adding empty fixtures to models/fixtures would probably be a good idea - then no one can make the mistake of expecting these things to be there across tests again.

I guess the question is if we can just empty the contentstore cheaply and easily in the prepareTestEnv.

Reply:
We do this already for the LFS files:
https://github.com/go-gitea/gitea/blob/6cb76bf1df581cf554670c533a1801b46a74909e/tests/test_utils.go#L201
so maybe `tests.PrepareTestEnv` would be a better place then? With the `storage.Clean` method the code would be shorter too because I don't need to get the package blobs.